src/main/java/com/schematical/chaoscraft/entities/EntityOrganism.java

```diff
@@ -101,6 +102,10 @@
     public Vec3d spawnPos;
     private boolean hasTraveled = false;
 
+    public boolean miningChat = false;
```

Comment:
Lets just check the static vars. Or better yet store it in ChaosCraftConfig -> https://github.com/schematical/chaoscraft-mod/blob/master/src/main/java/com/schematical/chaoscraft/ChaosCraftConfig.java

This way it persists even after it is reloaded.